bzr.go

```diff
@@ -24,6 +25,11 @@ func NewBzrRepo(remote, local string) (*BzrRepo, error) {
 	r.setRemote(remote)
 	r.setLocalPath(local)
 	r.Logger = Logger
+	
+	basePath := path.Join(r.LocalPath(), "..")
+	if _, err := os.Stat(basePath); err != nil {
+		err = os.MkdirAll(basePath, 0777)
+	}
```

Comment:
Three things stand out:

1. Why isn't `bzr` creating the directory?
2. A permission of `0777` is globally writable. That's not a good permission by default.
3. Any change like this should be in the `Get()` method instead of the constructor.